EventHighway.Core/Services/Processings/Events/V1/EventV1ProcessingService.cs

```diff
@@ -50,6 +50,14 @@ public ValueTask<IQueryable<EventV1>> RetrieveScheduledPendingEventV1sAsync() =>
                 eventV1.ScheduledDate < now);
         });
 
+        public ValueTask<EventV1> ModifyEventV1Async(EventV1 eventV1) =>
```

Comment:
Instead of building this function as pass through, why don't you make it do higher-order business logic as in: `MarkEventAsIntermediate`?

Reply:
got it. Thanks